packages/blade/tsconfig-typecheck.web.json

```diff
@@ -1,5 +1,11 @@
 {
   "extends": "./tsconfig-generate-types.web.json",
+  "exclude": [
```

Comment:
How does this work? Won't it exclude these sub extensions from typechecking 🤔

Reply:
Yes it will exclude all the files with these extensions `.stories.tsx`, `.native.ts` etc

Reply:
I didn't understand, it looks as if it would skip typechecking of these files, is that how it works? We want to typecheck everything

Reply:
Oh i see where you got confused.

So previously the `**.test.ts` files were being "excluded" from typechecking. 
The exclusions were coming from [tsconfig-generate-types.web.json](https://github.com/razorpay/blade/blob/master/packages/blade/tsconfig-generate-types.web.json#L4-L5) file. 

In this PR, I've redefined the "exclude" in `tsconfig-typecheck.web.json` and removed the `**test.ts` files.

Reply:
Lots of tsconfig files now, can we add some comments to these tsconfig files? I think tsconfig json supports comments:
- One comment at the top about the purpose of file
- Inline comment around exclude